### ISSUE TITLE
Incorrect optional chaining transpiling

### ISSUE THREAD

**Issue by starsolaris** [fuse-box/fuse-box#1892]
fuse-box@4.0.0-next.425
Next code:
```javascript
const a = {
    a: async () => {
        return "dd";
    }
};
let b = async () => {
    const b = await a?.a();
};
```

will be transpiled to:
```javascript
  const a = {
    a: async () => {
      return "dd";
    }
  };
  let b = async () => {
    const b = await (_1_ = a) === null || _1_ === void 0 ? void 0 : _1_.a();
  };
```

should be:
```javascript
const a = {
    a: async () => {
        return "dd";
    }
};
let b = async () => {
    const b = await ((_1_ = a) === null || _1_ === void 0 ? void 0 : _1_.a());
};
```
brackets after await is missing

**Comment by nchanged** [comment]
@starsolaris please verify the fix in `4.0.0-next.426`

**Comment by starsolaris** [comment]
@nchanged 4.0.0-next.426 works as expected.
